queries/cylc/highlights.scm

```diff
@@ -0,0 +1,104 @@
+(include_statement
+  directive: _ @keyword.directive
+  path: _ @string.special.path)
+
+(comment) @comment
+
+(graph_section
+  brackets_open: _ @punctuation.bracket
+  name: _? @label
+  brackets_close: _ @punctuation.bracket)
+
+(task_section
+  brackets_open: _ @punctuation.bracket
+  name: (_
+    (task_name) @function)
+  brackets_close: _ @punctuation.bracket)
+
+(top_section
+  brackets_open: _ @punctuation.bracket
+  name: _? @label
+  brackets_close: _ @punctuation.bracket)
+
+(sub_section_1
+  brackets_open: _ @punctuation.bracket
+  name: _? @label
+  brackets_close: _ @punctuation.bracket)
+
+(sub_section_2
+  brackets_open: _ @punctuation.bracket
+  name: _? @label
+  brackets_close: _ @punctuation.bracket)
+
+(runtime_section
+  brackets_open: _ @punctuation.bracket
+  name: _? @label
+  brackets_close: _ @punctuation.bracket)
+
+(graph_setting
+  key: (_) @number
+  operator: (_)? @operator)
+
+(quoted_graph_string
+  quotes_open: _ @string
+  quotes_close: _ @string)
+
+(multiline_graph_string
+  quotes_open: _ @string
+  quotes_close: _ @string)
+
+[
+  (graph_logical)
+  (graph_arrow)
+  (graph_parenthesis)
+] @keyword.operator
+
+(intercycle_annotation
+  (recurrence) @number)
+
+(graph_task
+  xtrigger: _? @keyword.modifier
+  suicide: _? @keyword.modifier
+  name: _ @function)
+
+(task_parameter
+  "<" @punctuation.bracket
+  name: (_)? @keyword.directive
```

Comment:
Would task parameter names make more sense as `@variable.parameter`? Genuine question, I don't know cylc at all

Reply:
task "parameters" in Cylc are basically used for templating. Less powerful than Jinja2 but less verbose. So I decided to use the same capture as Jinja2 nodes.
https://cylc.github.io/cylc-doc/latest/html/user-guide/writing-workflows/parameterized-tasks.html

Reply:
Sorry for the long delay. In this case I think maybe the jinja2 nodes and these should also be changed to `@variable.parameter`, or at least something besides `@keyword.directive` which doesn't make much sense to me

Reply:
@ribru17 So is there still something that should be changed in this PR?

Reply:
I think these should be changed to variable.parameter here. I can change jinja to match this in another pr (but do we even have jinja in this repo?)

Reply:
Do what you think is best. I won't edit this one though as it would discard the 3 approvals 😅 
As you are reviewers, maybe it won't if either of you does it ?